build.sbt

```diff
@@ -16,8 +16,8 @@ ThisBuild / tlFatalWarnings := false
 ThisBuild / tlBaseVersion := "4.5"
 
 ThisBuild / organization := "org.gnieh"
-ThisBuild / organizationName := "Lucas Satabin"
-ThisBuild / startYear := Some(2022)
+ThisBuild / organizationName := "Raphael Bosshard"
+ThisBuild / startYear := Some(2024)
```

Comment:
Sorry I wasn't clear I guess, please do not change these settings, they are global. I am in charge of managing them (and I need to update the year for sure).

Reply:
Then it's not quite clear to me how I can contribute to the project while retaining authorship and copyright to the contributed parts. Would you mind elaborating?

Reply:
Hi, authorship is given by the git history. As for the copyright I kindly ask you to give the project (non exclusive) Copyright for this contribution.

I will make it clearer in the `CONTRIBUTING` file and change the copyright to the project. My name is an artifact from the past, when it was a personal project, I will change to `diffson Project`.

Reply:
This works for me and my sponsor. :+1: